syntax = "proto3";

message EnterRoomC2S {
    
}